tests/cypress/support/commands.js

```diff
@@ -124,10 +122,12 @@ Cypress.Commands.add('switchLabel', (labelName) => {
 });
 
 Cypress.Commands.add('checkObjectParameters', (objectParameters, objectType) => {
-    cy.get('.cvat-objects-sidebar-state-item').then((objectSidebar) => {
-        cy.get(`#cvat_canvas_shape_${objectSidebar.length}`).should('exist').and('be.visible');
-        cy.get(`#cvat-objects-sidebar-state-item-${objectSidebar.length}`)
-        .should('contain', objectSidebar.length).and('contain', `${objectType} ${objectParameters.type.toUpperCase()}`).within(() => {
+    cy.get('.cvat_canvas_shape').last().should('have.attr', 'id').then(($cvatCanvasShapeId) => {
```

Comment:
If we use ``last()`` I am not sure it will work always because there exist different orderings. We can test them and this logic will be broken. To get exactly latest element we might choose an object with the highest ID. Also getting the latest element id could be a different cypress command.

Reply:
Changed to use the maximum value from ```#cvat_canvas_shape_```